tests/test_big_modeling.py

```diff
@@ -199,14 +207,14 @@ def test_cpu_offload(self):
 
         cpu_offload(model, execution_device=device)
         output = model(x)
-        assert torch.allclose(expected, output.cpu(), 1e-4, 1e-5), f"Expected: {expected}, Actual: {output.cpu()}"
+        torch.testing.assert_close(expected, output.cpu(), atol=ATOL, rtol=RTOL)
```

Comment:
Again error msgs

Reply:
in this case (and others `test_big_modeling`) isn't the output from `torch.testing.assert_close` enough ?
printing the entire tensors is too verbose no ?